### ISSUE TITLE
Can't map Delete to new key

### ISSUE THREAD

**Issue by jake-thewoz** [randyrants/sharpkeys#83]
Hello,

I found this awesome program after lots of googling. My Lenovo p50s had some keys lose functionality not long after I bought it- specifically Delete, Insert and PrtSc. I don't really care about Insert and PrtSc, but I need **Delete**. If I'm not hugely mistaken, the delete key is **E0-53**.

I've used SharpKeys to try to remap it to other keys with no success. First I tried the special calculator key; I remapped (E0-53) to it (E0-21), then assigned E0-21 to _Turned Off_. I wrote to the registry, several times, and rebooted. It successfully turned off the calculator function, but it did not add the Delete function.

I've tried this technique over several other keys with zero success. I've tried it with the Web key (E0-32) and the files key (E0-5B). I also tried not turning off the web/files/calculator key (the key I was moving E0-53 to), thinking maybe I was killing it. I've even tried it on the right asterisk key (00-37), same result.

Is there some deeper problem that might be causing these failures? Any guidance would be hugely appreciated. I've made it a couple years without using Delete, but now I really need it to log in to my old windows vms.

**Comment by randyrants** [comment]
I'm not sure what you're trying to to but if you remap E0_53 to E0_21 and then told E0_21 to be turned off, you've essentially turned off both keys.  Just remap E0_53 to E0_21 and that should be all you need.

**Comment by jake-thewoz** [comment]
SharpKeys just doesn't seem to work with E0-53 at all on my machine. I'm running Windows 10 on a Lenovo p50s. To verify, just now I mapped it to Right Ctrl (and wrote to the registry and rebooted), still nothing. I can turn off other keys, no problem. I haven't tried, but I'm sure I can remap other keys.

I even tried using Microsoft Keyboard Layout Creator 1.4 to make a custom keyboard where [del] is assigned to various keys, and none of it works. Perhaps there's some deeper problem with my firmware or something.

Fortunately I learned that I can send Ctrl+Alt+Del into a VM from VMWare itself, so my original problem isn't stopping me from my work anymore. It is still beyond frustrating that I can't seem to fix my broken delete key, no matter what I try. Thanks for the response.
